airbyte-integrations/connectors/source-qonto/source_qonto/spec.yaml

```diff
@@ -0,0 +1,28 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Qonto Spec
+  type: object
+  required:
+    - organization_slug
+    - secret_key
+    - iban
+  properties:
+    organization_slug:
+      type: string
```

Comment:
every item in properties/config needs to have a `title:` define for human readable format